### ISSUE TITLE
Just a question

### ISSUE THREAD

**Issue by stherrienaspnet** [db-migrate/mongodb#34]
Hello, I'm wondering if migrated scripts log can be stored into mongodb? I mean without using external file to track migrated script.

**Comment by wzrdtales** [comment]
I do not understand your question, db-migrate has no external files.

**Comment by stherrienaspnet** [comment]
So it mean, tracking of migrated script is done inside mongodb, if I correctly understand your answer

**Comment by wzrdtales** [comment]
yes, how do you get a doubt of any external files. There should be nothing even remotely indicating this?
